fastai/interpret.py

```diff
@@ -63,7 +81,10 @@ def plot_top_losses(self, k, largest=True, **kwargs):
         #its None means that a batch knows how to show itself as a whole, so we pass x, x1
         #else: show_results(x, x1, its, ctxs=ctxs, max_n=max_n, **kwargs)
 
-    def show_results(self, idxs, **kwargs):
+    def show_results(self,
+        idxs:(TensorBase,list), # Index of predictions and targets
```

Comment:
We are using list for any list-like object
```suggestion
        idxs:list, # Indices of predictions and targets
```